STACpopulator/models.py

```diff
@@ -0,0 +1,87 @@
+import datetime as dt
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import (
+    AnyHttpUrl,
+    AnyUrl,
+    BaseModel,
+    Field,
+    SerializeAsAny,
+    field_validator,
+)
+from typing_extensions import TypedDict
+
+
+class Geometry(TypedDict):
+    type: str
+    coordinates: List[List[List[float]]]
```

Comment:
Very specifc to polygon/multi-poligon. 
Should consider renaming the class if only this case is supported to avoid confusion. 
The `type` should also be `Literal[...]` of the specific types supported.